test/RankifyInstance.ts

```
@@ -1196,31 +1174,49 @@
+                  rankifyInstance.connect(adr.gameMaster1).endVoting(
                     1,
                     votes.map(v => v.vote),
-                    integrity.newProposals,
                     integrity.permutation,
                     integrity.nullifier,
                   ),
                 ).to.be.emit(rankifyInstance, 'TurnEnded');
               });
             });
-            describe('When first turn was made', () => {
+            describe.only('When first turn was made', () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)